config.go

```diff
@@ -89,7 +89,7 @@ type config struct {
 	DumpAllMPTix       bool   `long:"dumpallmptix" description:"Dump to file the fees of all the tickets in mempool."`
 	DBFileName         string `long:"dbfile" description:"SQLite DB file name (default is dcrdata.sqlt.db)."`
 
-	LiteMode bool   `short:"l" long:"lite" description:"Run in \"lite\" mode, using only SQLite."`
+	FullMode bool   `long:"full" description:"Run in \"Full Mode\" mode, using only SQLite."`
```

Comment:
description doesn't read right ("mode" repeated, and "using only SQLite" was for lite mode not full mode)